### ISSUE TITLE
Unable to install Vagrant development environment using Vagrantfile from the repo

### ISSUE THREAD

**Issue by czukowski** [hashicorp/vagrant#10484]
This is probably an issue in rmv install script that is used by the Vagrantfile with an added factor of an old Ubuntu version.

### Vagrantfile

https://github.com/hashicorp/vagrant/blob/master/Vagrantfile

### Output

Gist: https://gist.github.com/czukowski/8fd2dc50c231f7ca3239170ef7d2df42

Specifically this:

```
    default: Downloading https://github.com/rvm/rvm/archive/.tar.gz
    default: curl: (35) error:1407742E:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert protocol version
    default:
    default: Could not download 'https://github.com/rvm/rvm/archive/.tar.gz'.
    default:   curl returned status '35'.
    default: Downloading https://bitbucket.org/mpapis/rvm/get/.tar.gz
    default: curl: (35) error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure
    default:
    default: Could not download 'https://bitbucket.org/mpapis/rvm/get/.tar.gz'.
    default:   curl returned status '35'.
```

It is coming from the provision command:

```sh
# Install RVM
su -l -c 'curl -sL https://get.rvm.io | bash -s stable' vagrant
```

### Expected behavior

`vagrant up` completed successfully with rvm installed correctly.

### Actual behavior

rvm has not been installed and everything else depending on it as well.

### Steps to reproduce

1. `git clone https://github.com/hashicorp/vagrant.git .`
2. `vagrant up`

### Additional info

When trying the same steps, but with a newer Ubuntu (just replacing the box with `ubuntu/xenial64`), the installation has completed successfully, but the "latest" Ruby version detected by the provision script was `ruby-2.6.0-preview2` which was in conflict with the Gemfile:

```
    default: Resolving dependencies...
    default: Bundler could not find compatible versions for gem "ruby ":
    default:   In Gemfile:
    default:     ruby
    default:
    default:     vagrant was resolved to 2.2.3.dev, which depends on
    default:       ruby  (~> 2.2, < 2.6)
```

After installing Ruby 2.5.1 using rvm manually, all the tests comleted ok (3478 examples, 0 failures, 3 pending).

**Comment by daryl-sf** [comment]
Having the same issue. Did you manage to get it resolved?

**Comment by czukowski** [comment]
@daryl-sf  I changed the 'box' property to `ubuntu/xenial64` (randomly picked, many other will probably work too) and finished the provisioning manually by installing the latest Ruby 2.5.* at the time.

**Comment by briancain** [comment]
Closed via #11012
